dist/FormView.js

```diff
@@ -42,7 +42,7 @@
 
       //Attach Events to preexisting elements if we don't have a template
       if (!this.template) this.runInitializers();
-      this.on('item:rendered',this.runInitializers, this);
```

Comment:
@iamchrismiller any thoughts on getting this merged and creating the new version?